elixir/apps/web/test/web/controllers/auth_controller_test.exs

```diff
@@ -357,9 +357,9 @@ defmodule Web.AuthControllerTest do
           ~p"/#{provider.account_id}/sign_in/providers/#{provider.id}/request_magic_link",
           %{
             "email" => %{
-              "provider_identifier" => identity.provider_identifier,
-              "client_platform" => "platform"
-            }
+              "provider_identifier" => identity.provider_identifier
+            },
+            "client_platform" => "platform"
```

Comment:
@jamilbk did you test it manually? Because I remember that we create hidden inputs for those values so they will be part of the form submission, but don't remember exactly where the key was placed.

Reply:
Yes, the key is outside the `email`